integration/spark-common/src/main/scala/org/apache/carbondata/spark/util/CommonUtil.scala

```diff
@@ -54,6 +53,12 @@ object CommonUtil {
 
   val FIXED_DECIMAL = """decimal\(\s*(\d+)\s*,\s*(\-?\d+)\s*\)""".r
   val FIXED_DECIMALTYPE = """decimaltype\(\s*(\d+)\s*,\s*(\-?\d+)\s*\)""".r
+  val timestampFormatter = new SimpleDateFormat(CarbonProperties.getInstance
```

Comment:
SimpleDateFormat is not thread safe. 
In mutiple threads, it will return wrong value.